R/derive_params_growth_age.R

```diff
@@ -178,6 +190,7 @@
 #'   )
 #' )
 derive_params_growth_age <- function(dataset,
+                                     by_vars = exprs(USUBJID),
```

Comment:
don't we usually leave by_vars NULL as default in admiral? also dont hardcode USUBJID. you need to use get_admiral_option("subject_keys"). See this ongoing PR for examples https://github.com/pharmaverse/admiral/pull/2452

Reply:
@Fanny-Gautier FYI of this update as it is going to impact the template calls for this function (and i can update same in the vignette)